docs/extensions/guides/stores.md

```diff
@@ -42,18 +42,17 @@ export class ExamplePreferencesStore extends Store.ExtensionStore<ExamplePrefere
         enabled: false
       }
     });
+    makeObservable(this);
   }
 
   protected fromStore({ enabled }: ExamplePreferencesModel): void {
     this.enabled = enabled;
   }
 
   toJSON(): ExamplePreferencesModel {
-    return toJS({
```

Comment:
Wrapping with `toJS()` probably not necessary since:
- properly handled in `BaseStore.enableSync()` for reacting to every observable from `BaseStore.toJSON()`
- sanitizing payload for IPC-messaging (removing all possible observable objects from `BaseStore.toJSON()`)